Source/UnitTests/Common/SettingsHandlerTest.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <gtest/gtest.h>
+
+#include "Common/SettingsHandler.h"
+
+namespace
+{
+constexpr size_t BUFFER_SIZE = Common::SettingsHandler::SETTINGS_SIZE;
+
+// The encrypted bytes corresponding to the following settings, in order:
+//   "key" = "val"
+std::array<char, 9> BUFFER_CONTENTS_A = {'\x91', '\x91', '\x90', '\xEE', '\xD1',
```

Comment:
Done. (It's unfortunate that conversion of a character literal to `u8` is a narrowing conversion. Is there anything cleaner we can do here?)

Reply:
Rather than `'\x91'`, make it `0x91`. So hexadecimal integer rather than its char interpretation.

For the values that you can't easily convert by inspection ('B', '/', etc...), you can use a hex editor or Google to find its hexadecimal value.

Reply:
Ah right! Thanks.